fuzz/src/lib.rs

```diff
@@ -192,9 +192,12 @@ pub fn gen_config(node_ix: NodeIndex, n_members: NodeCount) -> Config {
         requests_interval: Duration::from_millis(50),
         unit_rebroadcast_interval_min: Duration::from_millis(400),
         unit_rebroadcast_interval_max: Duration::from_millis(500),
-        //100, 100, 300, 900, 2700, ...
-        unit_creation_delay: Arc::new(|t| exponential_slowdown(t, 50.0, usize::MAX, 1.000)),
         //50, 50, 50, 50, ...
+        unit_creation_delay: Arc::new(|t| exponential_slowdown(t, 50.0, usize::MAX, 1.000)),
```

Comment:
This is a strange way to write `Arc::new(|_| Duration::from_millis(50)),` -- I know it's some old code, but you can change it on this occasion :P